lib/miq_automation_engine/service_models/miq_ae_service_host.rb

```diff
@@ -21,6 +21,9 @@ class MiqAeServiceHost < MiqAeServiceModelBase
     expose :domain
     expose :files,                 :association => true
     expose :directories,           :association => true
+    expose :raw_set_node_maintenance
+    expose :raw_unset_node_maintenance
+    expose :raw_get_node_maintenance
```

Comment:
So I'm wondering if the right answer is more like the following:
1) Just drop the 'raw' prefix for the methods that actually make changes to the underlying OpenStack resource -- so set_node_maintenance instead of raw_set_node_maintenance, and unset_node_maintenance instead of raw_unset_node_maintenance
2) for raw_get_node_maintenance, change it to external_get_node_maintenance. The key here is that we need to distinguish between "go to OpenStack and report on the current setting" (so the "external_" one) and "return the activerecord attribute" -- I think it would be confusing to hide that difference.

With this, we lose the messy-looking "raw_" prefix while retaining clarity.